api/server/router/network/network_routes.go

```diff
@@ -85,6 +85,10 @@ func (n *networkRouter) getNetwork(ctx context.Context, w http.ResponseWriter, r
 	}
 
 	term := vars["id"]
+	verbose := false
+	if r.Form.Get("verbose") == "true" {
```

Comment:
The convention for our API endpoints that take boolean values seems to be to use `strconv.ParseBool`. This accepts alternate values such as `1`, `t`, `TRUE`, etc.